src/constants/triggers.ts

```diff
@@ -133,19 +133,19 @@ export const taskTriggers: Trigger = {
 // VERSION TRIGGERS
 export const versionTriggers: Trigger = {
   [VersionTriggers.VERSION_FINISHES]: {
-    trigger: TriggerType.OUTCOME,
+    trigger: TriggerType.FAMILY_OUTCOME,
     label: "This version finishes",
     resourceType: ResourceType.VERSION,
     payloadResourceIdKey: "id",
   },
   [VersionTriggers.VERSION_FAILS]: {
-    trigger: TriggerType.FAILURE,
+    trigger: TriggerType.FAMILY_FAILURE,
```

Comment:
Just for my understanding, is it true that the `family-` triggers only apply to entire versions? (i.e. nothing to do with tasks, build variants)

Reply:
yes, exactly! If someone wants to be notified when their entire patch is succeeded, failed, done, etc, then we only want to notify them when the entire patch AND the child patches are done.